File: par/zone/vue/src/view/com/drzyl/zone/zoneRegion/ZoneRegionQuery.js
export default class ZoneRegionQuery {
  constructor () {
    this.toolExists = null // 解决多对多关系预留字段之存在
    this.toolNotExists = null // 解决多对多关系预留字段之不存在
    this.regionId = null // ID
    this.parentId = null // 父ID
    this.regionName = null // 名称
    this.regionType = null // 类型
    this.agencyId = null // 代理
    this.status = null // 状态
    this.regionIdAndin = null // IDAndin查询
    this.parentIdAndKeyLike = null // 父IDAndKeyLike查询
    this.regionNameOrKeyLike = null // 名称OrKeyLike查询
    this.regionTypeOrKeyLike = null // 类型OrKeyLike查询
    this.agencyIdOrKeyLike = null // 代理OrKeyLike查询
    this.statusOrKeyLike = null // 状态OrKeyLike查询
  }
}

File: par/zone/zone-interface/src/main/java/com/drzyl/zone/query/ZoneRegionQuery.java
package com.drzyl.zone.query;

import java.io.Serializable;
import java.util.List;

import lombok.Data;

/**
 * 省市区 PAR_ZONE_REGION 查询类
 */
@Data
public class ZoneRegionQuery implements Serializable {

  private static final long serialVersionUID = 1L;

  protected String toolExists; // 解决多对多关系预留字段之存在

  protected String toolNotExists; // 解决多对多关系预留字段之不存在

  protected String regionId; // ID

  protected String parentId; // 父ID

  protected String regionName; // 名称

  protected String regionType; // 类型

  protected String agencyId; // 代理

  protected String status; // 状态

  protected List<String> regionIdAndin; // IDAndin查询

  protected String parentIdAndKeyLike; // 父IDAndKeyLike查询

  protected String regionNameOrKeyLike; // 名称OrKeyLike查询

  protected String regionTypeOrKeyLike; // 类型OrKeyLike查询

  protected String agencyIdOrKeyLike; // 代理OrKeyLike查询

  protected String statusOrKeyLike; // 状态OrKeyLike查询

}
